web/backend/src/cli.ts

```diff
@@ -65,4 +67,37 @@ program
     console.log('Permissions removed successfully!');
   });
 
+// Imports a list of SCIPERS from a file to allow to vote on a specific election
+// the .voters.example file is available as an example
+program
+  .command('addVoters')
+  .description('Assigns a list of SCIPERs to an Election as Voters')
+  .requiredOption('-e, --election-id <char>', 'ID of the election')
+  .requiredOption('-sf, --scipers-file <char>', 'File with line-separated list of SCIPERs')
+  .action(async ({ electionId, scipersFile }) => {
+    fs.readFile(scipersFile, 'utf8', async (err: any, data: string) => {
+      if (err) {
+        console.error(err);
```

Comment:
Didn't see this one, which should also `throw`.

Reply:
Yep, fixed as well.